packages/core/src/views/connect/Index.svelte

```diff
@@ -339,6 +340,19 @@
         })
       })
     }
+    if (
+      !secondaryTokens &&
+      Array.isArray(appChain.secondaryTokens) &&
```

Comment:
it's possible that appChain will be "undefined" if the chain selected in user's wallet doesn't exist in the config. This leads to an error:

> Cannot read properties of undefined (reading 'secondaryTokens')

:(

Reply:
@pavelivanov we have a patch coming out for this shortly - apologies for any inconvenience

Reply:
> we have a patch coming out for this shortly - apologies for any inconvenience

is this patch available?

Reply:
There is a fix for this on @web3-onboard/core v2.19.0-alpha.1 and an official release will be out shortly.